packages/sit-onyx/src/i18n/locales/ko-KR.json

```diff
@@ -1,20 +1,24 @@
 {
   "optional": "(선택사항)",
   "validations": {
-    "tooShort": "{minLength}자 이상으로 입력하세요. (현재 1글자 입력 중) | {minLength}자 이상으로 입력하세요. (현재 {n}글자 입력 중)",
-    "tooLong": "{maxLength}자 이하로 입력하세요. (현재 1글자 입력 중) | {maxLength}자 이하로 입력하세요. (현재 {n}글자 입력 중)",
-    "rangeUnderflow": "입력값이 {min}보다 크거나 동일해야 합니다",
-    "rangeOverflow": "입력값이 {max}보다 작거나 동일해야 합니다",
-    "patternMismatch": "올바른 형식을 입력해주세요.",
-    "valueMissing": "입력란에 내용을 입력해주세요.",
-    "stepMismatch": "{step}의 배수에 해당하는 값을 입력해주세요.",
-    "badInput": "\"{value}\"는 올바른 타입이 아닙니다.",
+    "tooShort": {
+      "fullError": "{minLength}자 이상으로 입력하세요. (현재 1글자 입력 중) | {minLength}자 이상으로 입력하세요. (현재 {n}글자 입력 중)"
```

Comment:
If only having fullErrors it will only show the tooltip, right and no error message?
I'd prefer to either remove the translations then or ask @nerdrun for updates ones :)

Reply:
you said the preview error will fall back to english

Reply:
I tried it in the form demo. that's how it behaves when korean is set:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/fbfe7740-fab3-447c-b93a-17ca30c75bcd)

Reply:
Ah true sorry^^